server/app.js

```diff
@@ -12,9 +15,12 @@ app.use(logger('dev'));
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 app.use(cookieParser());
-
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+app.use(cors());
 app.use('/', router);
 
 const server = app.listen(port, () => console.log(`Listening on port ${port}`));
+database.connectDB();
```

Comment:
Start db when server is connected, also i think this will return a promise, if yes then wait for it